### PR TITLE
[RFR] Improvements here and there

### PR DESCRIPTION
**Pull request by KittyGiraudel**
Hey @mattcav!

We are almost ready with the Italian version.

A few last things:
- I added some missing sentences in English; they need to be translated.
- I added the missing `it/_tldr.md` file that needs to be filled; it was missing entirely.

Feel free to merge this and keep working on your branch `mattcav-1.3.0`. It will automatically update https://github.com/HugoGiraudel/sass-guidelines/pull/355. :)

### CODE REVIEW

File: it/_mixins.md [hunk 1]
```diff
@@ -17,10 +17,7 @@ Un altro esempio valido potrebbe essere un _Mixin_ per impostare la grandezza di
 
 {% include snippets/mixins/02/index.html %}
 
-###### Approfondimenti
-
-* [Sass Mixins to Kickstart your Project](http://www.sitepoint.com/sass-mixins-kickstart-project/)
-* [A Sass Mixin for CSS Triangles](http://www.sitepoint.com/sass-mixin-css-triangles/)
+For more complex examples of mixins, have a look at [this mixin to generate CSS triangles](http://www.sitepoint.com/sass-mixin-css-triangles/), [this mixin to create long shadows](http://www.sitepoint.com/ultimate-long-shadow-sass-mixin/) or [this mixin to polyfill CSS gradients for old browsers](http://www.sitepoint.com/building-linear-gradient-mixin-sass/).
```
Comment (KittyGiraudel) [inline it/_mixins.md:20]: This sentence was missing. I added it in English, but it would need to be translated. :)

File: it/_naming.md [hunk 2]
```diff
@@ -27,9 +23,7 @@ Come per molti linguaggi, suggerisco di nominare le variabili costanti completam
 
 {% include snippets/naming/02/index.html %}
 
-###### Approfondimenti
-
-* [Dealing With Constants in Sass](http://www.sitepoint.com/dealing-constants-sass/)
+If you really want to play with the ideas of constants in Sass, you should read [this dedicated article](http://www.sitepoint.com/dealing-constants-sass/).
```
Comment (KittyGiraudel) [inline it/_naming.md:26]: This sentence was missing. I added it in English, but it would need to be translated. :)

File: it/_rwd.md [hunk 3]
```diff
@@ -31,13 +27,9 @@ Una volta che avrete dato il nome che preferite ai vostri _breakpoint_, avrete b
 
 <div class="note">
   <p>Ovviamente questo è un modo fin troppo semplice di gestire i <em>breakpoint</em>. Se avete bisogno di un metodo più permissivo, vi raccomando di non reinventare la ruota ed utilizzare qualcosa di testato ed efficace come: <a href="https://github.com/sass-mq/sass-mq">Sass-MQ</a>, <a href="http://breakpoint-sass.com/">Breakpoint</a> o <a href="https://github.com/eduardoboucas/include-media">include-media</a>.</p>
+  <p>If you are looking to read more on how to approach Media Queries in Sass, both <a href="http://www.sitepoint.com/managing-responsive-breakpoints-sass/">SitePoint</a> (from yours, truly) and <a href="http://css-tricks.com/approaches-media-queries-sass/">CSS-Tricks</a> have nice articles on this.</p>
```
Comment (KittyGiraudel) [inline it/_rwd.md:30]: This sentence was missing. I added it in English, but it would need to be translated. :)

File: it/_tools.md [hunk 4]
```diff
@@ -39,36 +33,19 @@ Se non siete legati ad uno specifico sistema di griglie, sarete contenti di sape
...
 
-
 [SCSS-lint](https://github.com/causes/scss-lint) è uno strumento che vi aiuta a mantenere i vostri file CSS puliti e leggibili. È totalmente configurabile ed è facile da integrare con i vostri strumenti.
 
 Fortunatamente, le raccomandazioni di SCSS-lint sono molto simili a quelle descritte in questo documento. Per configurare SCSS-lint secondo le queste linee guida, vi raccomando il seguente setup:
 
 {% include snippets/tools/01/index.html %}
 
+If you are not convinced of the necessity to use SCSS-lint, I recommend reading these great articles: [Clean Up your Sass with SCSS-lint](http://blog.martinhujer.cz/clean-up-your-sass-with-scss-lint/), [Improving Sass code quality on theguardian.com](http://www.theguardian.com/info/developer-blog/2014/may/13/improving-sass-code-quality-on-theguardiancom) and [An Auto-Enforceable SCSS Styleguide](http://davidtheclark.com/scss-lint-styleguide/).
```
Comment (KittyGiraudel) [inline it/_tools.md:46]: This sentence was missing. I added it in English, but it would need to be translated. :)

**Comment by mattcav** [comment]
Roger roger @HugoGiraudel. I will give it a look later.

**Comment by MakhBeth** [comment]
I've translated the missing sentences. 
I've done the first translation of _tldr, but I want to reread it :D

**Pull request merged**